pyproject.toml

```diff
@@ -31,6 +31,7 @@ disable = "missing-function-docstring,missing-module-docstring,line-too-long,too
 
 [tool.pytest.ini_options]
 minversion = "6.0"
+pythonpath = "."
```

Comment:
(Added this to be able to run `poetry run pytest` from the root folder.)

Reply:
Runs for me without setting pythonpath, regardless of activated venv or not

Reply:
Weird! As a workaround I can also do `PYTHONPATH=. poetry run pytest`, so I just removed this from the PR.